dist/tools/openocd/openocd.sh

```diff
@@ -93,6 +101,14 @@ test_elffile() {
     fi
 }
 
+test_elffile_antibrick() {
+    if [ -n "${ANTIBRICK}" ]; then
+        echo "Run ANTIBRICK check:"
+        echo "       (${ANTIBRICK})"
+        sh -c "${ANTIBRICK} '${ELFFILE}'" && exit 1
```

Comment:
The convention in shell scripts is to use `||` to handle errors and to return 0 on clean exit. `/bin/true` returns 0 and `/bin/false` return 1 as their exit codes.